lib/netext/httpext/batch.go

```diff
@@ -22,13 +22,10 @@ type BatchParsedHTTPRequest struct {
 // pre-initialized. In addition, each processed request would emit either a nil
 // value, or an error, via the returned errors channel. The goroutines exit when
 // the requests channel is closed.
-// The processResponse callback can be used to modify the response, e.g.
-// to replace the body.
 func MakeBatchRequests(
 	ctx context.Context, state *lib.State,
 	requests []BatchParsedHTTPRequest,
 	reqCount, globalLimit, perHostLimit int,
-	processResponse func(*Response, ResponseType),
```

Comment:
Open question; Is it fine to modify a publicly exposed function straight away? 🤔

Reply:
TBH, even if exposed, I'd argue that this is _real_ public k6's API-like modules, for example, so I think we're safe here. But at least we could mention this as a minor breaking change (adding the label)

Reply:
Sounds good to me, just asking from ignorance. Thanks!